### PR TITLE
DEP Use PHPUnit 11

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/305

### CODE REVIEW

File: tests/php/Page/SiteTreeTest_NamespaceMapTestNode.php
Comment (emteknetnz) [inline tests/php/Page/SiteTreeTest_NamespaceMapTestNode.php:8]: Because this file ends in "Test" - phpunit though it was a test class and complained that it had no test cases. So I simply renamed it to match its $table_name

File: tests/php/Controllers/SiteTreeTest_NamespaceMapTestNodeController.php
Comment (emteknetnz) [inline tests/php/Controllers/SiteTreeTest_NamespaceMapTestNodeController.php:8]: This was renamed to match the corresponding SiteTreeTest_NamespaceMapTestNode which was also renamed

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge once all other PRs are approved

**Pull request merged**
